patches/minecraft/net/minecraft/entity/LivingEntity.java.patch

```diff
@@ -531,12 +531,10 @@
 +   }
 +
 +   @Override
-+   public void remove(boolean keepData) {
-+      super.remove(keepData);
-+      if (!keepData) {
-+         for (int x = 0; x < handlers.length; x++)
-+            handlers[x].invalidate();
-+      }
++   protected void invalidateCaps() {
++      super.invalidateCaps();
```

Comment:
Why was the keepData check removed here?

Reply:
`invalidateCaps` for Entities is called from `Entity#remove` when `keepData` is false. This extra check got removed as `invalidateCaps` only will be called when it should be invalidated.